AudioSystem/AudioSystemExtension.cs

```diff
@@ -164,15 +164,15 @@ public static AudioSource Play(this AudioConfig audioConfig, float volume, Trans
 		/// <param name="onComplete">The callback when the playback of all configs finished</param>
 		/// <returns>The channel (AudioSource) which the Audio System is using for the first clip in the sequence</returns>
 		public static AudioSource PlaySequence(this IEnumerable<AudioConfig> audioConfigs, float volume = 1f, Transform parent = null, 
-			Func<int, int> getClipId = null, float delay = 0f, Action onComplete = null, Action<int> onEachConfigPlayed = null)
+			Func<int, int> getClipId = null, Func<int, float> getClipDelay = null, Action onComplete = null, Action<int> onEachConfigPlayed = null)
```

Comment:
With this many parameters I'd probably start to new line them. I'm not 100% sure if there is anything on this situation in the coding standard docs though.